erdjs/src/testdata/abi.json

```diff
@@ -0,0 +1,210 @@
+[
+    {
+        "namespace": "ultimate-answer",
+        "functions": [
+            {
+                "name": "getUltimateAnswer",
+                "input": [],
+                "output": [
+                    {
+                        "description": "the ultimate answer",
+                        "type": [
+                            "I64"
+                        ]
+                    }
+                ]
+            }
+        ],
+        "structures": []
+    },
+    {
+        "namespace": "counter",
+        "functions": [
+            {
+                "name": "increment",
+                "input": [],
+                "output": [
+                    {
+                        "description": "the current value",
+                        "type": [
+                            "I64"
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "decrement",
+                "input": [],
+                "output": [
+                    {
+                        "description": "the current value",
+                        "type": [
+                            "I64"
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "get",
+                "input": [],
+                "output": [
+                    {
+                        "description": "the current value",
+                        "type": [
+                            "I64"
+                        ]
+                    }
+                ],
+                "isPure": true
+            }
+        ],
+        "structures": []
+    },
+    {
+        "namespace": "lottery-egld",
+        "functions": [
+            {
+                "name": "start",
+                "input": [
+                    {
+                        "description": "lottery name",
+                        "type": [
+                            "Vector",
+                            "U8"
```

Comment:
This is equivalent to `Vector<U8>`, right?

Reply:
Yes.